nautobot/docs/configuration/required-settings.md

```diff
@@ -152,17 +152,20 @@ setup](https://github.com/Suor/django-cacheops#setup).
 
 ---
 
-### Task Queuing
+### Task Queuing with RQ
 
-Task queues are configured by defining them within the [`RQ_QUEUES`](#rq_queues) setting. 
+!!! warning
+	As of Nautobot 1.1 using task queueing with RQ is deprecated in exchange for using Celery. Support for RQ will be removed entirely starting in Nautobot 2.0.
+
+Task queues are configured by defining them within the [`RQ_QUEUES`](#rq_queues) setting.
 
 Nautobot's core functionality relies on several distinct queues and these represent the minimum required set of queues that must be defined. By default, these use identical connection settings as defined in [`CACHES`](#caches) (yes, that's confusing and we'll explain below).
 
-In most cases the default settings will be suitable for production use, but it is up to you to modify the task queues for your environment and know that other use cases such as utilizing specific plugins may require additional queues to be defined. 
+In most cases the default settings will be suitable for production use, but it is up to you to modify the task queues for your environment and know that other use cases such as utilizing specific plugins may require additional queues to be defined.
 
 #### CACHES
 
-The [`django-redis`](https://github.com/jazzband/django-redis) Django plugin is used to enable Redis as a concurrent write lock for preventing race conditions when allocating IP address objects, and also to define centralized Redis connection settings that will be used by RQ. The `CACHES` setting is required to to simplify the configuration for defining queues. *It is not used for caching at this time.*
+The [`django-redis`](https://github.com/jazzband/django-redis) Django plugin is used to enable Redis as a concurrent write lock for preventing race conditions when allocating IP address objects, and also to define centralized Redis connection settings that will be used by RQ and Celery. The `CACHES` setting is required to to simplify the configuration for defining queues. *It is not used for caching at this time.*
```

Comment:
It's a bit confusing that this section is under `Task Queueing with RQ` now, if it applies to both RQ and Celery?

Reply:
Yeah I missed that, thanks.